### PR TITLE
feat: send bitcoin back navigation

### PR DESCRIPTION
**Pull request by UncleSamtoshi**
- separate send-bitcoin flow into separate screens, so that it can navigate back during the flow and facilitate deeplinking

### CODE REVIEW

**Review by daviroo — COMMENTED** [review]
Just wanted to highlight the commented out code for now.  I'll need to check out the branch and test it locally before I can approve.

**Comment by UncleSamtoshi** [comment]
This Pr addresses issues #549 #558 #536 #557

**Review by daviroo — CHANGES_REQUESTED** [review]
I've made some suggestions and left some comments.

File: app/screens/send-bitcoin-screen/send-bitcoin-success-screen.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+const SendBitcoinSuccessScreen = ({
+  navigation,
+}: StackScreenProps<RootStackParamList, "sendBitcoinSuccess">) => {
+  const { refetch } = useMainQuery()
+  const CALLBACK_DELAY = 2000
+  useEffect(() => {
+    refetch()
+    setTimeout(() => navigation.popToTop(), CALLBACK_DELAY)
+  }, [])
```
[the remark this answers is not in the document]
Comment (UncleSamtoshi) [reply app/screens/send-bitcoin-screen/send-bitcoin-success-screen.tsx:44]: Great catch!

File: app/screens/send-bitcoin-screen/scanning-qrcode-screen.tsx [hunk 2]
```diff
@@ -99,18 +100,16 @@ export const ScanningQRCodeScreen: ScreenType = ({
           if ("reason" in lnurlParams) {
             throw lnurlParams.reason
           }
-
+          // May not be neccessary any more
```
[the remark this answers is not in the document]
Comment (UncleSamtoshi) [reply app/screens/send-bitcoin-screen/scanning-qrcode-screen.tsx:103]: I'll remove it. The comment was to indicate that some of the code in the scanningQRScreen is duplicated in the sendBitcoinDestination screen. However, this is out of scope.

**Review by daviroo — APPROVED** [review]

**Pull request merged**